server/tokens.go

```diff
@@ -308,9 +310,18 @@ func (c *Server) blindedTokenRedemptionHandler(w http.ResponseWriter, r *http.Re
 		}
 
 		if issuer.Version == 2 {
-			redemption, err := c.fetchRedemptionV2(issuer, tokenID)
+			issuerUUID, err := uuid.FromString(issuer.ID)
+			if err != nil {
+				c.Logger.Error("Bad issuer id")
```

Comment:
Are these errors worth the `error` level or can we make them more debug level?

Reply:
Worth noting that pretty much everything from the AppError gets logged when the handler returns so all the info should be there for prod debugging. If we were doing something which would hide the internal error from the client then it would be better to log like this once when the error comes back.

```
	logger.
		WithField("issuerID", issuer.ID).
		WithError(err).
		Error("bad request for id")
	
	...
	
	handlers.AppError{
		Message: fmt.Sprintf("Bad request for issuerID %s", issuer.ID),
		Code:    http.StatusBadRequest,
	}
```